locksmith/examples/websub-discord/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "websub-discord",
   "packageManager": "yarn@3.1.0-rc.7",
   "dependencies": {
+    "@unlock-protocol/networks": "workspace:./packages/networks",
```

Comment:
Hum since we would then this example to be standalone (and I assume reproducible) let's not use the workspace here but hardcode some values in a config file?

Reply:
Is it possible to deploy @unlock-protocol/networks package to npm?

Other people might benefit from it too and we can use that directly here.

Reply:
I don't want to copy over the networks package to example because that mean maintaining two copies of the same. 

If we could publish networks package to npm, we could use @unlock-protocol/networks directly from npm in subsequent PR in future to make it completely standalone.